phpBB/styles/prosilver/template/posting_poll_body.html

```diff
@@ -43,6 +43,7 @@
 				<dd><label for="poll_vote_change"><input type="checkbox" id="poll_vote_change" name="poll_vote_change"{VOTE_CHANGE_CHECKED} /> {L_POLL_VOTE_CHANGE_EXPLAIN}</label></dd>
 			</dl>
 		<!-- ENDIF -->
+		<!-- EVENT posting_poll_body_options_after -->
```

Comment:
This gets me learning how to use git with pull requests :P Hope it is correct now.